documentation/CONFIG_OUTLIERS.md

```diff
@@ -319,16 +323,84 @@ run_model=1
 test_model=0
 ```
 
+## Sudden Appearance models
+The sudden_appearance model looks for outliers by finding the sudden appearance of a certain field(s).
+Example use case: tag sudden appearance of a new TLD DNS. 
```

Comment:
Change this into "the sudden appearance of a website that is being visited, that has never been visited in the past by a specific user or computer".